components/catalogue/categoryEditForm/GroupList.tsx

```diff
@@ -64,7 +62,7 @@ const GroupList = () => {
             {fields.map((field, index) => (
               <li key={field.id} className="flex py-2 ">
                 <Group
-                  removeGroup={removeGroup}
```

Comment:
You could have just kept `removeGroup={remove}` and it would work the same way without changing the interface etc.